eureka2-client/src/main/java/com/netflix/eureka2/client/channel/InterestChannelImpl.java

```diff
@@ -113,9 +117,11 @@ public void call(Subscriber<? super Void> subscriber) {
                 } else if (moveToState(STATE.Idle, STATE.Open)) {
                     logger.debug("First time registration: {}", newInterest);
                     channelInterestStream.subscribe(channelInterestSubscriber);
+                    remoteBatchingRegistry.subscribe(channelInterestStream);
```

Comment:
Since there is ever only one channelInterestStream, we should be able to attach it to the remoteBatchingRegistry when the channelInterestStream is created in the constructor and not need to reattach it each time we do a change().